### DISCUSSION TITLE
How to disable the temporary duplicated newline with new input for each character change?

### DISCUSSION THREAD

**Discussion by trondhe** [wezterm/wezterm#2844] [Q&A]
My google-foo and lua-reference looking has failed me and I cannot figure out how to remove this duplicating behaviour.

Its a bit noisy for me and I would like this behaviour to be disabled. Any tips to a wezterm newbie?

**Comment by bew** [comment]
That looks like a bug.. I'd suggest showing a video of what you are seeing

**Reply by trondhe** [reply]
Opening a new terminal window and started writing. Nushell has a lot of "startup messages" so it fills the whole window
Inputting
```
qwertyuio
```
![image](https://user-images.githubusercontent.com/7525911/206179532-5f7a1375-7d9a-43f9-951b-6caf8116b293.png)

---
If I run clear (nushell), then it does not have this behaviour
Inputting
```
clear<ret>
123456....
```
![image](https://user-images.githubusercontent.com/7525911/206179632-05f8c241-e458-42cb-9e61-4b452e417324.png)

---


As a third try I inputted
```
clear<ret>
<ret>
afdafg...
```
![image](https://user-images.githubusercontent.com/7525911/206179883-578f46ca-8a8a-4b51-acff-8e8664f990cd.png)
It does this repeating behaviour only on these two visible lines

---


Doing first four empty newlines, it repeats this behaviour
```
clear<ret>
<ret>
<ret>
<ret>
abcdefg...
```
![image](https://user-images.githubusercontent.com/7525911/206180062-639f6045-2a7c-4d60-a65e-dc0d39dab958.png)


edit: updated comment to be more concise/readable

**Reply by bew** [reply]
That definitely looks like a bug.
I'd suggest you to open an issue to report this bug, and give all information about your environment, version of nushell, screenshots...
Do you see this behavior on other terminals? (like on Windows Terminal?) If so that should probably be an issue on nushell's repo.

**Reply by wez** [reply]
Please search the open issues: there is already one open about this nushell problem

**Reply by trondhe** [reply]
I didnt notice it was a bug since it looks actually "ok" although noisy as long as the terminal has been filled entirely from the getgo.
Makes sense I didnt find it then.

This looks to have somewhat similar behaviour https://github.com/wez/wezterm/issues/2779
